build.rs

```diff
@@ -0,0 +1,102 @@
+use serde::{Deserialize, Serialize};
+use std::{fs, io::Write};
+use toml::de;
```

Comment:
Learning for me, how are the crates referenced? I.E. cards, game, scoring, etc.?

Reply:
Spoke about this one in teams.